articles/communication-services/how-tos/router-sdk/handle-offer.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Handle an offer in Job Router
+titleSuffix: An Azure Communication Services how-to guide
+description: Use Azure Communication Services SDKs to manage the job offer
+author: marcma
+ms.author: marcma
+ms.service: azure-communication-services
+ms.topic: how-to 
+ms.date: 01/18/2022
+ms.custom: template-how-to
+
+#Customer intent: As a developer, I want to handle job offers when coming in.
```

Comment:
I think this should be more focused on accept/decline, "Handle" is too ambiguous.